### PR TITLE
Models

### PR DESCRIPTION
**Pull request by zujko**

### CODE REVIEW

**Comment by snoozan** [comment]
That commit history is messy. Can you do a git reset --soft and then roll all your commits together or into a few?

**Comment by aaron13michael** [comment]
I don't see the benefits of clumping all the commits into a few and giving
a false impression of the commit history. Also can we be more professional
than describing things that we don't like as 'messy AF'? It's disrespectful.

**Comment by snoozan** [comment]
My logic for rolling the commits together is because there seemed to be many small commits that don't warrant their own messages.

File: main/models.py [hunk 1]
```diff
@@ -1,3 +1,61 @@
...
+class Property(models.Model):
+    address = models.CharField(max_length=120)
+    city = models.CharField(max_length=100)
+    state = models.CharField(max_length=2)
+    zip_code = models.CharField(max_length=5)
+    latitude = models.DecimalField(max_digits=9, decimal_places=6)
+    longitude = models.DecimalField(max_digits=9, decimal_places=6)
+    furnished = models.BooleanField()
+    rooms = models.PositiveSmallIntegerField()
+    complx = models.ForeignKey(Complex)
```
Comment (NathanCastle) [inline main/models.py]: Will this allow for properties that don't have a complex?

Comment (zujko) [inline main/models.py]: This did not, I have pushed a new commit that allows a property to not have an associated complex.

**Comment by NathanCastle** [comment]
Looks good, feel free to merge at your convenience.

**Pull request merged**